<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceQuestionWidgetClass</class>
 <widget class="QWidget" name="ChoiceQuestionWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChoiceQuestionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame{
border-radius:10px ;
background-color: rgba(0,0,0,0.4);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Question</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="questionText">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>here comes the text of the choice question</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="halveOptions">
        <property name="text">
         <string>Halve options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="option1">
        <property name="text">
         <string>Option1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="option2">
        <property name="text">
         <string>Option2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="option3">
        <property name="text">
         <string>Option3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="option4">
        <property name="text">
         <string>Option4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="timerProgressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
